The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/controllers/DestinationController.java

```diff
@@ -17,10 +18,24 @@ public class DestinationController {
     @Autowired
     private DestinationService destinationService;
 
+    @Autowired
+    private GeocodingService geocodingService;
+
     @PostMapping
-    public ResponseEntity<Destination> createDestination(@RequestBody Destination destination) {
-        Destination createdDestination = destinationService.saveDestination(destination);
-        return ResponseEntity.status(HttpStatus.CREATED).body(createdDestination); //201 response
+    public ResponseEntity<Destination> createDestination(@RequestParam String name) {
+
+        try {
+            System.out.println("TESTING");
```

Comment:
Let's remove this print statement before merging into main

Reply:
removed! thank you!